README.md

```diff
@@ -75,6 +75,12 @@ or description of the image. Imgur can host some really nasty things, so
 this seems to be a required feature.
 
 
+#### nsfw
+
+By default, files marked as nsfw are ignored, but you can change this by
```

Comment:
Optional but add a link to http://www.urbandictionary.com/define.php?term=NSFW&defid=248838 ? haha.